src/components/biggive-table/biggive-table.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'biggive-table',
+  styleUrl: 'biggive-table.scss',
+  shadow: false,
+})
+export class BiggiveTable {
+  /**
+   * Space below component
+   */
+  @Prop() spaceBelow: number = 0;
+
+  /**
+   * Header text colour
```

Comment:
One thing that it would be useful to document imho, whether it's via code or docblock is what sort of colour strings these accept. Maybe it could be by using a union type of all accepted colours, or just a description in the docblock.